<launch>
  <!-- <param name="use_sim_time" value="true" /> -->

  <node	pkg="sdf_slam_2d" type="sdf_slam" name="sdf_slam_2d" output="screen">
    <param name="p_timeout_ticks_" value="000" />	    
    <param name="p_update_map_aoe_" value="5" />	    
    <param name="p_dist_threshold_" value="0.4" />	    
    <param name="p_yaw_threshold_" value="0.9" />	    

    <param name="p_vanity_" value="0" />	    

    
    <param name="p_initial_pose_" type="bool" value="true" />	    
    <param name="p_publish_map_" type="bool" value="false" />	    

    <param name="p_avg_mapping_" type="bool" value="true" />	    
    <param name="p_fine_pos_" type="bool" value="true" />	    
    <param name="p_perma_map_" type="bool" value="false" />	    

    
    <param name="p_robot_frame_" type="string" value="robot0" />	    
    <param name="p_fixed_frame_" type="string" value="map" />	    
    <param name="p_tar_frame_" type="string" value="sdf_pos" />	    


    <param name="p_min_range_" value="0.0" />	    

    <remap from="scan" to="robot0/laser_0" />
  </node>

</launch>
